spec/spec.md

```diff
@@ -142,6 +172,104 @@ Examples in this document use the Verifiable Credentials Data Model [[spec:VC-DA
   }
 }
 ```
+::: Entity Oriented Example
+
+This pulls the topic to the top of the document, and indexes properties by entity.
```

Comment:
Is this supposed to say "This pulls the entity to the top of the document and indexes properties by topic"?